frontend/components/queries/PackQueriesTable/PackQueriesTable.tsx

```diff
@@ -90,14 +90,20 @@ const PackQueriesTable = ({
             })
           }
           showMarkAllPages={false}
-          actionButtonText={"Add query"}
-          actionButtonIcon={AddQueryIcon}
-          actionButtonVariant={"text-icon"}
-          onActionButtonClick={onAddPackQuery}
-          onPrimarySelectActionClick={onRemovePackQueries}
-          primarySelectActionButtonVariant="text-icon"
-          primarySelectActionButtonIcon="close"
-          primarySelectActionButtonText={"Remove"}
+          actionButton={{
```

Comment:
Action buttons and primary select actions are now written as objects for better organization and easier readability